pkg/p2p/libp2p/internal/handshake/handshake.go

```diff
@@ -214,6 +224,8 @@ func (s *Service) Handle(stream p2p.Stream, remoteMultiaddr ma.Multiaddr, remote
 	}
 
 	s.logger.Tracef("handshake finished for peer %s", remoteBzzAddress.Overlay.String())
+	s.logger.Tracef("received welcome message %s from peer", ack.WelcomeMessage)
```

Comment:
This message is in the Handle method, meaning that it will be printed on the node that is connected to. I think that the logging should happen on the node that is requesting a connection. WTDY?

This should be logged only if welcome message is not empty.

I think that Info log level would be better for visibility of this message, as Info is the default one.

Maybe to log also the peer overlay address at the end of the log message? @Eknir any suggestions on this?

Reply:
I agree with what you write here. It should definitely be in the info. And maybe you can make it more active:

peer <overlay_address> greets you: <welcome_message>

Reply:
> This message is in the Handle method, meaning that it will be printed on the node that is connected to. I think that the logging should happen on the node that is requesting a connection. WTDY?

I agree, I will remove this when the PR is ready - just wanted to have a trace on both sides of the handshake when tracing.